style.css

```diff
@@ -156,3 +175,17 @@ header {
 .task-container .task-done {
     background-color: var(--background-color-done);
 }
+
+
+footer ul {
+    background-color: #a88957;
```

Comment:
please use the defined css variable, --background-color-footer, to set the background-color. With that, it woud be much easier for us to fine rune the theme and color.